cassandra/src/main/scala/filodb.cassandra/columnstore/ShardKeyToPartKeyTable.scala

```diff
@@ -0,0 +1,63 @@
+package filodb.cassandra.columnstore
+
+import scala.concurrent.{ExecutionContext, Future}
+
+import com.datastax.driver.core.{ConsistencyLevel}
+import monix.reactive.Observable
+
+import filodb.cassandra.FiloCassandraConnector
+import filodb.core.{DatasetRef, Response}
+
+sealed class ShardKeyToPartKeyTable(val dataset: DatasetRef,
+                                    val shard: Int,
+                                    val connector: FiloCassandraConnector,
+                                    writeConsistencyLevel: ConsistencyLevel)
+                                   (implicit ec: ExecutionContext) extends BaseDatasetTable {
+
+  import filodb.cassandra.Util._
+
+  val suffix = s"shardKeyToPartKey_$shard"
+
+  // TODO(a_theimer): compression settings okay?
+  // TODO(a_theimer): probably need to prevent hot-spotting on a single node
+  //   (i.e. add more fields to [Cassandra's] partition key)
```

Comment:
I won't do this until benchmark tests indicate we need it (or one of you already knows we'll need it).